great_expectations/compatibility/sqlalchemy_and_pandas.py

```diff
@@ -38,18 +37,14 @@ def execute_pandas_reader_fn(
     Returns:
         dataframe or list of dataframes
     """
-    if is_version_less_than(pd.__version__, "2.0.0"):
-        if sqlalchemy != SQLALCHEMY_NOT_IMPORTED and is_version_greater_or_equal(
-            sqlalchemy.__version__, "2.0.0"
-        ):
+    if sqlalchemy and is_version_less_than(pd.__version__, "2.0.0"):
+        if is_version_greater_or_equal(sqlalchemy.__version__, "2.0.0"):
             warn_pandas_less_than_2_0_and_sqlalchemy_greater_than_or_equal_2_0()
         with warnings.catch_warnings():
             warnings.filterwarnings(action="ignore", category=RemovedIn20Warning)
             reader_fn_result: pd.DataFrame | list[pd.DataFrame] = reader_fn(
                 **reader_options
             )
     else:
-        reader_fn_result: pd.DataFrame | list[pd.DataFrame] = reader_fn(
-            **reader_options
-        )
```

Comment:
@anthonyburdi please check my logic here but this should fix the error we get in the quickstart when SQLAlchemy isn't installed

Reply:
I've reverted this stuff